vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -141,7 +141,7 @@ public class PieceDefiner extends JPanel {
   private JLabel scaleLabel;
   private final Prefs prefs;
 
-  private String prototypeName; // If we're editing a prototype definition, this holds the name of it
+  private static String prototypeName; // If we're editing a prototype definition, this holds the name of it
```

Comment:
Why change this to static? It's possible to have multiple Piece Definers open at the same time, If you open a piece, then a prototype before closing the piece, the line numbers in the piece start acting like a prototype

Reply:
> Why change this to static? It's possible to have multiple Piece Definers open at the same time, If you open a piece, then a prototype before closing the piece, the line numbers in the piece start acting like a prototype

Thanks Brent. I suspected there'd be an issue but lack of Java knowledge meant this was the quickest workaround for me. For the next build I am now using an independent means of determining if a Prototype is being processed.

PS I was unable to replicate problem behaviour to see what it looked like but I've just realised I didn't use the precise sequence of opening that you indicated above.